modules/check_in/app/services/travel_claim/claim_submission_service.rb

```diff
@@ -383,19 +396,30 @@ def error_template_id
     def determine_error_template_id(error)
       if error.is_a?(Common::Exceptions::BackendServiceException) &&
          error.response_values[:detail]&.include?('already been created')
-        @facility_type&.downcase == 'oh' ? CheckIn::Constants::OH_DUPLICATE_TEMPLATE_ID : CheckIn::Constants::CIE_DUPLICATE_TEMPLATE_ID
+        template_for_duplicate_error
       else
         error_template_id
       end
     end
 
+    ##
+    # Returns duplicate template ID based on facility type
+    #
+    def template_for_duplicate_error
+      if @facility_type&.downcase == 'oh'
+        @context_constants::OH_DUPLICATE_TEMPLATE_ID
+      else
+        @context_constants::CIE_DUPLICATE_TEMPLATE_ID
```

Comment:
same comment as below for the metrics but regarding the templates, we probably wouldn't want to name the new app's template constants `CIE_DUPLICATE_TEMPLATE_ID` and `OH_DUPLICATE_TEMPLATE_ID`